pantheon/src/main/java/tech/pegasys/pantheon/cli/PublicKeySubCommand.java

```diff
@@ -110,4 +113,51 @@ public void run() {
       }
     }
   }
+
+  /**
+   * Public key address export sub-command
+   *
+   * <p>Export of the public key address takes a file as parameter to export directly by writing the
+   * address in the file. A direct output of the address to standard out is not done because we
+   * don't want the address value to be polluted by other information like logs that are in
+   * KeyPairUtil that is inevitable.
+   */
+  @Command(
+      name = "export-address",
+      description = "This command exports the node's public key address to a file.",
+      mixinStandardHelpOptions = true)
+  static class AddressSubCommand implements Runnable {
+
+    @Option(
+        names = "--to",
+        required = true,
```

Comment:
Suggestion:  AFAIK: "--to" is not a standard name for output file.  If you agree, consider using "-o / --out" instead as I think it is more common.

Reply:
but it's in our cli guidelines and how other commands works so we have to be consistant.

Reply:
--to is our CLI guideline.

I would recommend making it not required and in it's absence outputting to System.out.

Reply:
I'm in agreement with @shemnon. Using --to and if it's not specified output to the console.

Reply:
--to is now optional. I also made the change on the other one. It's retro-compatible with the previous way to use the commands.